### PR TITLE
Helper headers for interop copy

### PR DESCRIPTION
**Pull request by 9prady9**
These headers are for external libraries or applications that
use forge for their interop visulizations.

### CODE REVIEW

File: include/fg/CudaCopy.hpp [hunk 1]
```diff
@@ -0,0 +1,59 @@
+/*******************************************************
```
Comment (shehzan10) [inline include/fg/CudaCopy.hpp]: Can you change the filename to CUDACopy

File: include/fg/CPUCopy.hpp [hunk 2]
```diff
@@ -0,0 +1,43 @@
+/*******************************************************
```
Comment (shehzan10) [inline include/fg/CPUCopy.hpp]: Move these headers into include/ not include/fg. fg should contain only internal headers.

Comment (9prady9) [inline include/fg/CPUCopy.hpp]: but these functions are under fg namespace and it makes sense to leave them under fg folder.

File: include/fg/CudaCopy.hpp (hunk 1 above)
Comment (9prady9) [inline include/fg/CudaCopy.hpp]: sure

File: include/fg/CPUCopy.hpp (hunk 2 above)
Comment (shehzan10) [inline include/fg/CPUCopy.hpp]: We can have them in the fg namespace without keeping them in the fg folder.
I don't think these count as forge headers, ie. not included in include/forge.h. Hence I believe they should be outside.

Comment (9prady9) [inline include/fg/CPUCopy.hpp]: ok, i am fine with moving them out. Just saying that 'include' is also forge header location so i don't understand your point.

**Comment by 9prady9** [comment]
Updated it with changes. ready for merge.

**Pull request merged**